### PR TITLE
Created Contact Screen, added FAQ text and relevant socials, added theme

### PR DESCRIPTION
**Pull request by r6097**
I added a 'Contact Us' button so that someone can implement the contact form later. I made 'theme.js' so that we can have consistent styling/coloring across the screens.

### CODE REVIEW

File: screens/ContactScreen.js [hunk 1]
```diff
@@ -1,10 +1,66 @@
...
+			</TouchableOpacity>
+
+			<View style={{ flexDirection:'row', justifyContent:'space-between', paddingHorizontal: 60, paddingBottom: 10 }}>
+				<TouchableOpacity
+				    onPress={() => Linking.openURL('https://www.instagram.com/mind_design_sports/')}
+				>
+					<AntDesign name="instagram" size={40} color="black" />
+				</TouchableOpacity>
+				<TouchableOpacity
+				    onPress={() => Linking.openURL('https://www.linkedin.com/company/mind-design-sports/')}
```
Comment (RSadaq) [inline screens/ContactScreen.js:43]: Twitter & LinkedIn urls need to be swapped over as currently twitter icon has linkedIn url & linkedIn icon has twitter url.

Comment (RSadaq) [inline screens/ContactScreen.js]: Twitter & LinkedIn urls need to be swapped over as currently twitter icon has linkedIn url & linkedIn icon has twitter url.

**Review by RSadaq — CHANGES_REQUESTED** [review]
Looks good so far. Are you able to style the paragraph on ContactScreen.js (e.g. more spacing, maybe add an image)?  Thanks.

File: screens/ContactScreen.js (hunk 1 above)
Comment (r6097) [reply screens/ContactScreen.js]: Addressed the changes in a new commit

**Review by RSadaq — APPROVED** [review]
Much better. Great job!

**Pull request merged**